chore: running the dynatrace behind a corporate proxy (#122)
Dynatrace/backstage-plugin: plugins/dql-backend/README.md

The backend can be started locally using `yarn start`, this starts the backend
on [localhost:7007/dql](http://localhost:7007/dql).

## Behind a corporate proxy

> If you are application is behind a corporate proxy. set HTTP_PROXY as
> environment variable. It will pass to HttpsProxyAgent in fetch request.

Reviewer: ```suggestion
If your application is behind a corporate proxy, set `HTTP_PROXY` as
an environment variable. This will be passed to `HttpsProxyAgent` as `agent` in the fetch request.
```

Author: done
